superset/assets/src/explore/components/ExploreChartPanel.jsx

```diff
@@ -1,8 +1,7 @@
-import $ from 'jquery';
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Panel } from 'react-bootstrap';
-
+import ParentSize from '@vx/responsive/build/components/ParentSize';
```

Comment:
FYI I don't think the deep import is necessary because `vx` exports es modules and has `sideEffects: false` in the `package.json` which should be all that webpack 4 needs for tree-shaking.

No harm in keeping this, tho.

Reply:
Oh sure. I was mindlessly copying from dashboard. Will change that one as well then.